zsh/zshrc

```diff
@@ -1,259 +1,35 @@
-# Profiling via:
-# https://kev.inburke.com/kevin/profiling-zsh-startup-time/
+# Portable Zsh configuration
+# This file sources modular configuration files that gracefully handle missing dependencies
+
+# Profiling support (keep at top for accurate measurements)
 : "${PROFILE_STARTUP:=false}"
 : "${PROFILE_ALL:=false}"
-# Run this to get a profile trace and exit: time zsh -i -c echo
```

Comment:
Put back all the comments in this profiling section

Reply:
you might want to check what's on main here to see what was commeted out before, I don't remember